Cargo.toml

```diff
@@ -4,7 +4,17 @@ categories = ["gui"]
 description = "Cross-platform window creation library."
 documentation = "https://docs.rs/winit"
 edition.workspace = true
-exclude = ["/.cargo"]
+include = [
+    "/build.rs",
+    "/docs",
+    "/examples",
+    "/FEATURES.md",
+    "/LICENSE",
+    "/src/**/*.rs",
+    "/src/changelog/*.md",
+    "/src/platform_impl/web/script/**/*.min.js",
```

Comment:
Can't we just include all of the `src`?

Reply:
I changed it up a bit to handle the TS/JS stuff explicitly.